packages/rest-specs/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@typespec/rest-specs",
+  "version": "0.37.2",
+  "description": "Spec scenarios and mock apis",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "watch": "tsc -p ./tsconfig.build.json --watch",
+    "build": "tsc -p ./tsconfig.build.json",
+    "clean": "rimraf dist/ temp/",
+    "validate-scenarios": "spec validate-scenarios ./http",
+    "generate-scenarios-summary": "spec generate-scenarios-summary ./http",
+    "upload-manifest": "spec upload-manifest ./http --storageAccountName azuresdkcadlranch",
+    "upload-coverage": "spec upload-coverage --storageAccountName azuresdkcadlranch",
+    "validate-mock-apis": "spec validate-mock-apis ./http",
+    "check-scenario-coverage": "spec check-coverage ./http",
+    "serve": "spec serve ./http",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/typespec.git"
+  },
+  "author": "Microsoft",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/microsoft/typespec/issues"
+  },
+  "homepage": "https://github.com/microsoft/typespec#readme",
+  "dependencies": {
+    "@typespec/spec": "workspace:*",
+    "@typespec/spec-api": "workspace:~",
+    "multer": "^1.4.5-lts.1",
+    "axios": "^1.7.5",
+    "dotenv": "~16.4.5"
+  },
+  "devDependencies": {
+    "@azure-tools/typespec-autorest": "0.46.0",
+    "@azure-tools/typespec-azure-resource-manager": "0.46.0",
+    "@azure-tools/typespec-client-generator-core": "0.46.0",
+    "@types/node": "~22.5.4",
+    "@types/multer": "^1.4.10",
+    "@typespec/openapi": "workspace:~",
+    "@typespec/openapi3": "workspace:~",
+    "rimraf": "~6.0.1",
+    "typescript": "~5.6.2"
+  },
+  "peerDependencies": {
+    "@typespec/spec-lib": "workspace:~",
+    "@azure-tools/typespec-azure-core": "~0.46.0",
```

Comment:
remove here too

Reply:
It is used in 6 services. Do you really want to remove all of them?

Reply:
Yes, which ones? If they are using azure core it means they are doing azure only things

Reply:
@timotheeguerin Here are the services that use:

1. payload/pageable
2. server/versions/versioned
3. type/enum/fixed
4. model/inheritance/enum-discriminator
5. property/additional-properties
6. property/value-types

Reply:
As mentioned in another comment, all references to that library has been removed.